assertk/src/commonMain/kotlin/assertk/assertions/iterable.kt

```diff
@@ -73,6 +74,40 @@ fun Assert<Iterable<*>>.containsOnly(vararg elements: Any?) = given { actual ->
     }.toString())
 }
 
+/**
+ * Asserts the iterable contains exactly the expected elements, in any order.
```

Comment:
Think this doc could be more descriptive in describing the different between this and the other contains methods.